src/components/LetterInputs/LetterInputs.js

```diff
@@ -118,69 +109,35 @@ function LetterInputs() {
               maxLength={1}
               key={i}
               value={letterState.letter}
-              ref={element => inputRefs.current[i] = element}
+              ref={(element) => (inputRefs.current[i] = element)}
               onChange={(event) => updateLetterState(i, event.target.value)}
               className={
                 submitted ? (letterState.status ? "correct" : "incorrect") : ""
               }
             />
           ))}
         </div>
-        {/* {counterValue === 10 ? (
-        // ==== UPDATE =====
-        // I think that this actually won't be a 'button' with a link - we'll need to just immediately route to the results page IF the counter value increments to 10
-        // Not sure what that will mean for code but I think below code will change. <--- Agreed! Made the change. 
-        <Link to="/results">
-          <button
-            className="submit-word-btn"
-            onClick={() => {
-              const guessedWord = letterStates.join("");
-              console.log(guessedWord);
-              dispatch(update(guessedWord));
-              dispatch(increment());
-            }}
-          >
-            Submit
-          </button>
-        </Link>
-      ) : ( */}
-        {/* // ===== UPDATE END ===== */}
-        <button
-          className='submit-word-btn'
-          onClick={handleSubmission}
-          //NOTE: Moved the below logic to a handle
-          // onClick={() => {
-          //   if (isWordCorrect(letterStates)) {
-          //     // If all the letters are correct send true to update reducer -> goes to the next word
-          //     dispatch(update(true));
-          //     dispatch(increment());
-          //     setLetterStates(
-          //       letterStates.map((letterState) => ({
-          //         letter: letterState.letter,
-          //         status: false,
-          //       }))
-          //     );
-          //   }
-          //   // what happens if false? Nothing? Maybe highlight the letters that have status: false
-          // }}
-        >
+        <button className="submit-word-btn" onClick={handleSubmission}>
           submit
         </button>
       </div>
-      {/* )} */}
-      <div className='feedback-container'>
-        {/* we will need to add logic here to display a descriptive message if right or wrong */}
-      {incorrectCount ? (
-        <h2 className='feedback-message'>You've got this! You're {incorrectCount} letter off!</h2>
-      ) : (
-        <h2 className='feedback-message'></h2>
-      )
-      }
+      <div className="feedback-container">
+        {incorrectCount === 1 ? (
+          <h2 className="feedback-message">
+            You've got this! You're {incorrectCount} letter off!
+          </h2>
+        ) : incorrectCount > 1 ? (
+          <h2 className="feedback-message">
+            So close! You're {incorrectCount} letters off!
```

Comment:
Love this attention to detail!  Isn't it wild how easy it is to miss things like this that seem so obvious in retrospect?